code_wars/check_same_case.py

```diff
@@ -1 +1,10 @@
 # https://www.codewars.com/kata/5dd462a573ee6d0014ce715b
+def same_case(a, b):
+    case_result = 2
+    if isinstance(a, str) == True:
+        if a.isupper() == True and a.isupper() == b.isupper():
+            return case_result == 1
+            if a.isupper() == False and a.isupper() == b.isupper():
+                return case_result == 0
```

Comment:
Really you don't need 2 checks here, and the logic is a bit off.

If both characters are the same case, return 1
but not the same case, return 0

So what do you think about:
```python
if a.isupper() == b.isupper():
    return 1
else:
    return 0
```

Return early with -1 before letting python read this far if either isn't a letter at all.